### PR TITLE
add use web api

### PR DESCRIPTION
**Pull request by nahumtimerman**
cloudshell server can now communicate if server is using "Web API" end point. This is useful for customers who want to take advantage of better performance at high load.
connectivity context will now send an indication that is using web api endpoint.

### CODE REVIEW

File: cloudshell/shell/core/driver_context.py [hunk 1]
```diff
@@ -65,13 +68,15 @@ def __init__(
...
         cloudshell_api_scheme,
+        use_webapi_endpoint=None,
     ):
         self.server_address = server_address
         self.cloudshell_api_port = cloudshell_api_port
         self.quali_api_port = quali_api_port
         self.admin_auth_token = admin_auth_token
         self.cloudshell_version = cloudshell_version
         self.cloudshell_api_scheme = cloudshell_api_scheme
+        self.use_webapi_endpoint = use_web_api_endpoint
```
Comment (saklar13) [inline cloudshell/shell/core/driver_context.py:79]: in the params and here different variable names - use_web_api_endpoint and use_webapi_endpoint (underscore in web api)

File: cloudshell/shell/core/driver_context.py [hunk 2]
```diff
@@ -55,6 +55,9 @@ class ConnectivityContext:
     :type cloudshell_version: str
 
     :type cloudshell_api_scheme: str
+
+    :param use_webapi_endpoint: Is Cloudshell API Web Api Endpoint enabled on server?
```
Comment (saklar13) [inline cloudshell/shell/core/driver_context.py:59]: As I understand the value can be "True" or "False", right? Can you mention this in the comment, please

**Review by saklar13 — CHANGES_REQUESTED** [review]

File: cloudshell/shell/core/driver_context.py (hunk 1 above)
Comment (nahumtimerman) [reply cloudshell/shell/core/driver_context.py]: thank you so much!!!
will now build the package and test to be sure it works properly

File: cloudshell/shell/core/driver_context.py (hunk 2 above)
Comment (nahumtimerman) [reply cloudshell/shell/core/driver_context.py]: good point!

**Comment by nahumtimerman** [comment]
@saklar13 ok, I made some changes, and also tested locally, LMK if ok

**Review by saklar13 — APPROVED** [review]

**Pull request merged**